examples/caching/with-nextjs-13/.env.local.example

```
@@ -1,2 +1,2 @@
 NEXT_PUBLIC_SUPABASE_URL=
-SUPABASE_SERVICE_ROLE_KEY=
\ No newline at end of file
+SUPABASE_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)